### ISSUE TITLE
Hello, can you add the following content to the reamme.md file

### ISSUE THREAD

**Issue by yangma12** [uds-lsv/MCSE#1]
Hello, can you add the following content to the reamme.md file:
1. Where should the pre-trained model file be placed in the code base after downloading.
2. Can you add script commands to run the test set?
Thank you for taking the time to check my question, hope to get your reply

**Comment by mrzhang11** [comment]
Hi, you can find some demo scripts in scripts/.  for both training and evaluation. To specify a model, you can use the _--model_name_or_path_ argument.
